src/napari/layers/shapes/_accelerated_triangulate_numba.py

```diff
@@ -876,7 +876,8 @@ def reconstruct_polygons_from_edges(
                 # chain of vertices, without this break, we would enter an
                 # infinite loop. Therefore, we leave it here for safety.
                 break
-        polygons.append([vertices[x] for x in poly])
+        polygon = [vertices[x] for x in poly]
```

Comment:
poly is List(int)
polygons is supposed to be `list of np.ndarray`
Can we do
```suggestion
        polygon = vertices[np.array(poly)]
```